### PR TITLE
Stylistic cleanup

### PR DESCRIPTION
**Pull request by mambrozio**
This closes issue #19. Did not enforce 80-column limit on a handful of cases which would look too bad if wraped. Changed coder.lua only when renamed variables was needed because Hugo intends to rewrite that file anyway.

### CODE REVIEW

File: pallene/driver.lua [hunk 1]
```diff
@@ -143,8 +145,8 @@ end
 -- This is meant for unit tests.
 --
 function driver.test_ast(stop_after, input_filename)
-    local basename, err = check_source_filename("pallenec test", input_filename, "pallene")
-    if not basename then return false, {err} end
+    local base_name, err = check_source_filename("pallenec test", input_filename, "pallene")
```
Comment (gligneul) [inline pallene/driver.lua]: 80 chars.

File: pallene/lexer.lua [hunk 2]
```diff
@@ -70,15 +71,15 @@ do
...
     local contents = (-matching_close * P(1)) ^0
 
-    longstring = (
+    long_string = (
         open * (
             C(contents) * close +
             T("UnclosedLongString")
         )
-    ) / function(contents_str) return contents_str end -- hide the group captures
+    ) / function(contents_str) return contents_str end --hide the group captures
```
Comment (gligneul) [inline pallene/lexer.lua]: I'd rather undo this one.

File: pallene/location.lua [hunk 3]
```diff
@@ -26,45 +26,45 @@ local function binary_search(xs, v)
     return lo
 end
 
-local newline_cache = setmetatable({}, { __mode = "k" })
+local new_line_cache = setmetatable({}, { __mode = "k" }) --new_line_cache?
```
Comment (gligneul) [inline pallene/location.lua]: It is together: https://en.wikipedia.org/wiki/Newline.

File: pallene/syntax_errors.lua [hunk 4]
```diff
@@ -55,8 +55,8 @@ local errors = {
     AssignImport = "Expected '='.",
 
     --this label is not thrown in rule 'import' because rule 'toplevelvar'
-    --matches an input like "local bola = X", given that X is a valid expression,
-    --or throws a label when X is not a valid expression
+    --matches an input like "local bola = X", given that X is a valid
```
Comment (gligneul) [inline pallene/syntax_errors.lua]: Let's add spaces after the `--` here.

**Comment by hugomg** [comment]
Thanks for the quick review, Gabriel. I agree with your comments.

I volunteer to make the required tweaks. I want to get this merged sooner than later so I can start rebasing the rest of my code on top...

File: pallene/syntax_errors.lua (hunk 4 above)
Comment (mambrozio) [reply pallene/syntax_errors.lua]: Here I just followed the original comment. Do I change all the comments in this file to add spaces after all the `--`?

File: pallene/lexer.lua (hunk 2 above)
Comment (mambrozio) [reply pallene/lexer.lua]: If undone, 80 chars will be exceed in the line. Do I move the comment to the line before or after? Or leave it with 81 chars?

**Comment by mambrozio** [comment]
Sorry about the useless comments. I thought I took them all out. I will make sure the next time. Agree with the others, just had some comments/doubts on a couple of them.

**Pull request merged**